modules/installation-special-config-butane.adoc

```diff
@@ -0,0 +1,128 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-special-config-butane_{context}"]
+= Creating machine configs with Butane
+
+Machine configs are used to configure control plane and worker machines by instructing machines how to create users and file systems, set up the network, install systemd units, and more.
+
+Because modifying machine configs can be difficult, you can use Butane configs to create machine configs for you, thereby making node configuration much easier.
+
+[id="rhcos-about-butane_{context}"]
+== About Butane
+
+Butane is a command-line utility that {product-title} uses to provide convenient, short-hand syntax for writing machine configs, as well as for performing additional validation of machine configs. The format of the Butane config file that Butane accepts is defined in the
+https://coreos.github.io/butane/specs/[OpenShift Butane config spec].
```

Comment:
Rather than specify the OpenShift 4.8.0 spec, I went up a dir-level to keep it green.